ntbs-service/wwwroot/source/Components/ConditionalSelectWrapper.ts

```diff
@@ -0,0 +1,34 @@
+import Vue from "vue";
+
+const ConditionalSelectWrapper = Vue.extend({
+    props: {
+        valueConditionFunction: {
+            type: Function
+        }
+    },
+    methods: {
+        innerValidateMounted: function () {
+            const value = this.getValueFromChild();
+            this.hideOrShowControlBasedOnValue(value);
+        },
+        getValueFromChild: function () {
+            if (this.$refs["inner-validate"] && this.$refs["inner-validate"].$refs["selectField"])
+                return this.$refs["inner-validate"].$refs["selectField"].value;
+            return null;
```

Comment:
I'd be tempted to add `{}` to the if:
- general good practice of avoiding changing branching by adding a line of code
- it's already visually busy, so a bit more structure might actually help